package.json

```diff
@@ -43,6 +43,7 @@
   "devDependencies": {
     "chai": "3.5.0",
     "coffee-19-jshint": "2.3.1",
+    "coffee-jshint": "0.2.3",
```

Comment:
Ok but first, let's ask to @aenario why he chosed this package over the official one.

Whatever, we should only have one package.